docs/GITOPS-CONFIG.md

```diff
@@ -20,31 +20,43 @@ If you installed using the [`ai-rhdh-installer`](../README.md#install) all that
 ### AI-RHDH-Installer: Manual Configuration
 
 #### Step 1: ConfigMaps and Secrets
-In your chosen namespace you should apply the following ConfigMaps:
-1.  [`argocd-config.yaml` ](../resources/argocd-config.yaml)
-2.  [`argocd-plugins.yaml`](../resources/argocd-plugins.yaml)
+In your chosen namespace you should apply the `argocd-config` ConfigMap under [`argocd-config.yaml` ](../resources/argocd-config.yaml).
 
 As part of the `ai-rhdh-installer` a secret was created in your desired namespace with the name `rhdh-argocd-secret`, keep note of this Secret as well as the two ConfigMaps applied above.
```

Comment:
> Here I think it gets confusing. Installing with `ai-rhdh-installer` is mentioned as pre-requisite for the script configuration. Reading this sentence makes me think it should be for both right?

@thepetk I'm not sure I fully understand the question, `ai-rhdh-installer` would be a prerequisite for all configure scripts as the configure scripts are ran after the fact, the only exception this is not the case is if the user already brought their own components in which case they need to follow the 'Pre-Existing Instance' steps.

Reply:
Yeah I believe there might have been some confusion, in this case it is the manual configuration for tekton if you installed the resources using the `ai-rhdh-installer` so it would be a prereq, like @michael-valdron said the `pre-existing instances` would have other options available!

Reply:
In the wording of the content I suppose you're right but the sections are prefixed with the method, **`AI-RHDH-Installer`**, no?